airbyte-config/config-models/src/main/resources/types/JobResetConnectionConfig.yaml

```diff
@@ -49,9 +44,6 @@ properties:
     existingJavaType: io.airbyte.config.ResourceRequirements
   resetSourceConfiguration:
     "$ref": ResetSourceConfiguration.yaml
-  state:
-    description: optional current state of the connection
-    "$ref": State.yaml
```

Comment:
source/destination configuration and state are removed from JobResetConnectionConfig